napalm_yang/parser.py

```diff
@@ -70,8 +70,10 @@ def _execute_methods(self, device, methods):
             if isinstance(r, dict) and all([isinstance(x, (str, unicode)) for x in r.values()]):
                 # Some vendors like junos return commands enclosed by a key
                 r = "\n".join(r.values())
-
-            result.append(r)
+            if isinstance(r, list):
+                result += r
```

Comment:
could you do instead `result.extend(r)`? I think it's easier to read,

Reply:
Fixed